### ISSUE TITLE
Cask installed Duet Display app fails to launch 

### ISSUE THREAD

**Issue by timup** [caskroom/homebrew-cask#14952]
```bash
~
$ brew cask install duet
==> Downloading http://dl.devmate.com/com.kairos.duet/1.4.0/1439081188/duet-1.4.
Already downloaded: /Library/Caches/Homebrew/duet-1.4.0_1439081188.zip
==> Symlinking App 'duet.app' to '/Applications/duet.app'
🍺  duet staged at '/opt/homebrew-cask/Caskroom/duet/1.4.0_1439081188' (355 files, 10M)

~
$ cd /applications

/applications
$ open duet.app
```

And then, nada.

Downloading the app from the duet display site, you are greeted with the following message after opening the app initially:

![Imgur](http://i.imgur.com/kuEIj4a.jpg?1)

Is it possible that this authorization is missed/bypassed somehow by the cask install?

Running El Cap, so maybe a new issue?

**Comment by adityadalal924** [comment]
Are you sure `duet.app` is in /applications?

Please make sure that `/Applications/duet.app` actually exists, but otherwise please give us some more more information, as described in [Reporting Bugs](https://github.com/caskroom/homebrew-cask#reporting-bugs)

**Comment by reitermarkus** [comment]
I just tested this. A symlink in `/Applications` doesn't work, because the app tries to copy itself to `/Applications` without prompting and if a symlink exists, it fails.

**Comment by vitorgalvao** [comment]
Another one for https://github.com/caskroom/homebrew-cask/issues/12858. We’ll need to add a caveat.

**Comment by timup** [comment]
Can confirm @reitermarkus.  Opening from ~/downloads copies the app to /applications before the permission prompt.

**Comment by vitorgalvao** [comment]
Closing in favour of https://github.com/caskroom/homebrew-cask/issues/13201, which will fix this. At the moment we don’t have any other type of workaround that could be used, anyway.

**Comment by toonetown** [comment]
Is there a way to suppress putting in the symlink?  I just got bitten by this one - `brew cask install duet` will automatically place the link in /Applications/duet.app - but then the first time you run it, it tries to copy itself /Applications/duet.app, but cannot because the link already exists.  Then it gets in this cycle of crashing and trying to restart.

This is, however, because I have `HOMEBREW_CASK_OPTS=--appdir=/Applications` set...so it is definitely an edge case in my setup.

If we could specify *not* to create a symlink, however, then it should work...since the first time it runs, it will be copied into the correct location.  I do agree, however, that a caveat should be added.

**Comment by vitorgalvao** [comment]
@toonetown See #13201 and https://github.com/caskroom/homebrew-cask/pull/13966.

**Comment by toonetown** [comment]
@vitorgalvao Yes - I had been following those discussions previously.  I was just wondering if there was a "quick fix" (specify to skip linking phase completely).  It's not a huge issue right now for me...I've got the app installed now ;)

Would it be helpful to add a caveat section now? (I can do that, if it would be helpful)  Or would it be better to just wait on the #13966 and #13201 to be implemented?

**Comment by vitorgalvao** [comment]
It’s best to wait for it to be implemented. Hopefully not long, now.

**Comment by toonetown** [comment]
I have no problem with that...just wanting to help out wherever I can!
